core/src/test/resources/test-transactions.conf

```diff
@@ -0,0 +1,89 @@
+{
+  entities: {
+    event: {
+      attributes: {
+        data: { scheme: "bytes" }
+      }
+    }
+
+    gateway: {
+      attributes: {
+        status: {
+          scheme: bytes
+          transactional: attribute
+          manager: gateway-transaction-commit-log
+        }
+
+        # the following defines a pattern for attributes
+        # each attribute that matches the pattern is treated the same
+        "device.*": {
+          scheme: bytes
+          transactional: entity
+          manager: gateway-transaction-commit-log
+        }
+      }
+    }
+
+    user {
+      manager: [ user-transaction-commit-log-request, user-transaction-commit-log-response, user-transaction-commit-log-state ]
```

Comment:
Manager of what? I think we should be here more precise. Maybe use structure like
```
transactions {
  type: "entity"
  request: <AF>
  response: <AF>
  state: <AF>
}
```
WDYT?

Reply:
The problem is that you can have multiple attribute families for request and/or response for the same attribute (of entity `_transaction`). The word `manager` might not be self-explanatory, but refers to the design document https://docs.google.com/document/d/1yPKIczGfi8tit3U89OJS9wEtCc8nIhL6VI6c35lYVhw/edit?usp=sharing
There is M:N relation between transactional attribute family and transactional attribute, which makes it hard to write it down to the config, it requires indirect references like this.

Reply:
The reason why this relation is M:N is that although the attribute family can (of course) contain only single entity (and possibly multiple attributes), the `request` and `response` can (indirectly) refer any other entity or attribute. The reason why there isn't only one attribute family for all transactional requests and/or responses is that that would mean only single consistency level - that would have to be global probably, thus boiling the parallelism down to 1. That is undesirable.

Reply:
One more note, we *could* use something like
```
 user {
   transactions {
     type: "..."
     request: AF
     response: AF
     state: AF
   }
 ```
but few problems:
 a) what if different attributes of the same entity have different transaction type?
 b) what if request, response and state attributes share the same family? That will be pretty much common, because you actually need to split that only on very, very, high volume workloads